src/Filters/Block.php

```diff
@@ -964,4 +967,69 @@ private function preparePriceSpecifications()
             'currencySymbol' => $currency->sign,
         ];
     }
+
+    /**
+     * Get the reviews filter block
+     *
+     * @param array $filter
+     * @param array $selectedFilters
+     *
+     * @return array
+     */
+    private function getReviweBlock($filter, $selectedFilters)
+    {
+        $values = [
+            '4' => ['name'=>"4"],
+            '3' => ['name'=>"3"],
+            '2' =>  ['name'=>"2"],
+            '1' => ['name'=>"1"],
+        ];
+
+        $query = 'SELECT  count(g.grade) as count ,g.grade FROM (SELECT t.id_product, case 
+when t.avg_grade between 1 and 1.99 then "1"
```

Comment:
We shouldn't be able to do 0.5 star? I think it costs nothing to start at 0 instead of 1

Reply:
actually the average score can not be less than 1, because we do not have comments with 0 grade, user must choose between 1-5 values.